source/access/mfa_guide.rst

```diff
@@ -208,27 +195,61 @@ on your local machine first.
 
 .. _mfa_client_config:
 
+GUI applications with SSH connection in the background
+------------------------------------------------------
+
+Some applications such as MobaXTerm and FileZilla provide a Graphical User Interface
+(GUI) which makes them very useful when connecting to remote sites. However,
+such an application may not always prompt you to copy/paste the VSC firewall link to
+set up the necessary SSH certificate.
+
+Therefore, one way to connect to VSC is to first connect with an ssh-client 
+on your machine as explained in the previous sections.
+In the case of not having an SSH certificate agent running then as long as you
+keep that connection open you can connect with the other apps as well.
+In case you have already stored the SSH certificate in a running agent
+you can then proceed with connecting to VSC with the application.
+
+.. note::
+
+   Some GUI applications may not always work when connecting to VSC
+   without an agent. For example, NX is one of them.
+   Therefore, it is highly recommended in such cases to use the agent
+   connection method :ref:`ssh agent<mfa_agent>`.
+
 Configuration of ssh-clients and GUI apps
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you have not yet set up your ssh-client or other apps that use ssh to be
 able to use an agent, you might have to make some changes in your connection
-profiles. Different apps will need different changes, but here we shortly show
-what to do for MobaXTerm, PuTTY and NX:
+profiles. Different apps will need different changes. For an explanation
+with screenshots for the various GUI apps please refer to :ref:`mfa quick start`.
+Here below we shortly show what to do for PuTTY, MobaXTerm, NX, and FileZilla:
+
+- PuTTY
+
+  - Under 'Session' in the tree menu enter the remote hostname. The port is usually 22.
+  - Under 'Connection/SSH' ensure the protocol is set to SSH.
+  - Under 'Connection/SSH/Auth' ensure that using Pageant and agent forwarding are active.
+  - Under 'Connection/SSH/Auth/Credentials' make sure that there are no private keys specified
+    in the 'Private key file for authentication' field.
+  - Go back to 'Session' and save your newly created session.
+  - Now you can load and open it to connect to VSC.
 
 - MobaXTerm
```

Comment:
There is a dedicated page for MobaXterm: https://docs.vscentrum.be/access/access_using_mobaxterm.html
I have not gone through that page (specially at the bottom where it talks about Moba's internal SSH agent - being Pageant). but, maybe what you have here is already there. Best is to refer to that page, if it covers our MFA case.

Reply:
Yes, same situation here as with filezilla. In general, a lot of connection/mfa pages need to be revisited. We only need to make sure we do not have wrong info for Tuesday.

Reply:
The issue with that page is that it still talks about SSH keys. Same with the Putty pages. I will only put tag KUL for now.